clion-cmake-test.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+mkdir -p cmake-build-debug && cd cmake-build-debug
+cmake ..
+cmake --build . --target tests
```

Comment:
it works, generally; I'm just wondering why you use `cd` and then `cmake ..`

```suggestion
mkdir -p cmake-build-debug
cmake .
cmake --build cmake-build-debug --target tests
```

Reply:
this is just an aesthetic question - in the end, it doesn't really matter

Reply:
> Have you considered adding an option which also makes the cache?
> In another project, I check if the cmake-build-debug folder exists, and if not it generates the cache. That way, it works from the first time on, and makes it easy to delete the whole cache (which sometimes is necessary)

that project went a bit out of control; sometimes I oversee things which are too simple XD

Reply:
(also, --build by safely be replaced by a -B, if you want shortness)

Reply:
cmake still confuses me

Reply:
Yeah I wasn't quite aware of how this works - I was merely regularly annoyed to find cmakes generated files in the project root ^^